plugins/notifications-frontend/src/components/NotificationsTable/MarkAsUnreadIcon.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles({
+  svg: {
+    width: 'auto',
+    height: 24,
+  },
+  path: {
+    fill: '#7df3e1',
+  },
+});
+
+const MarkAsUnreadIcon = () => {
```

Comment:
Not sure what is the backstage take on creating new icons, @freben @Rugvip ?

Reply:
I was just unable to find a pair of `read` and `unread` icons. Suggestions welcome.

Reply:
Perhaps use something from the MUI icons library, there's plenty to choose from